### ISSUE TITLE
Feature: Apply tags to sub folders/files

### ISSUE THREAD

**Issue by cjfagerstrom** [files-community/Files#12937]
### What feature or improvement do you think would benefit Files?

I would like to be able to apply a tag to a folder and have the option to apply the tag to all sub folders and/or sub files.

I have folders organized by company I work with and personal projects. I would like to be able to quickly locate files in the folders by tags, but I have thousands and files in a number of folders and it would save hours (days...weeks) to be able to apply tags in a cascading fashion like this.

### Requirements

- Right click on a tag that has been applied to a folder and be provided a short menu of options
- Menu will provide the following options: Apply tag to subfolders, Apply tag to subfiles, Apply tag to both 

### Files Version

2.5.10.0

### Windows Version

Windows 11 Pro Version	22H2 OS build	22621.1928 Windows Feature Experience Pack 1000.22644.1000.0

**Comment by Josh65-2201** [comment]
Thanks for the feedback, This could be added with #11992

**Comment by yaira2** [comment]
Are you looking for the tags to automatically be applied if you add more items later on?

**Comment by yaira2** [comment]
Thanks for the response, is this an option you'd like to apply any time you add a tag or is it something you'd like to be able to choose each time?

**Comment by yaira2** [comment]
@cjfagerstrom

**Comment by cjfagerstrom** [comment]
Each time.

**Comment by Cinabutts** [comment]
Agreed, was gonna write my own feature request asking the exact thing. i was hoping a simple command palette setting though, e.g `Add tag to file/folder and subfolders/files` or something like that.
I'm on windows 11 25h2 if that makes a difference btw.
